backend/src/main/java/com/google/sps/servlets/AddParticipantServlet.java

```diff
@@ -43,17 +37,6 @@
 @WebServlet("/api/v1/add-participant")
 public class AddParticipantServlet extends HttpServlet {
 
-  // Datastore Key/Property constants
-  private static final String KEY_PARTICIPANT = "Participant";
-  private static final String KEY_MATCH = "Match";
-  private static final String PROPERTY_USERNAME = "username";
-  private static final String PROPERTY_STARTTIMEAVAILABLE = "startTimeAvailable";
-  private static final String PROPERTY_ENDTIMEAVAILABLE = "endTimeAvailable";
-  private static final String PROPERTY_DURATION = "duration";
-  private static final String PROPERTY_TIMESTAMP = "timestamp";
-  private static final String PROPERTY_FIRSTPARTICIPANT = "firstParticipant";
-  private static final String PROPERTY_SECONDPARTICIPANT = "secondParticipant";
-
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
I was wondering what your thoughts were on separating the functionality of the add-participant servlet? From my read of this method I see both it posting a participant to participant datastore and searching for an immediate match. I know we talked about this earlier when the datastores and matching algorithm were structured a little differently, but I was wondering whether or not there's still a benefit to trying to find an immediate match

Reply:
I think there definitely still is a benefit to trying to find an immediate match because it ensures that no two participants are ever compared multiple times.